src/main/python/janusgraph_python/driver/ClientBuilder.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ..structure.io.GraphsonReader import JanusGraphSONReader
+from ..structure.io.GraphsonWriter import JanusGraphSONWriter
+
+from gremlin_python.driver.driver_remote_connection import DriverRemoteConnection
+from gremlin_python.structure.io.graphsonV3d0 import GraphSONReader
+from gremlin_python.structure.io.graphsonV3d0 import GraphSONWriter
+
+
+class JanusGraphClient(object):
+    """ JanusGraph Client Builder which adds the Serializers for JanusGraph specific objects, predicates etc. """
+
+    REMOTE_CONNECTION = None
+
+    def __init__(self, version=3.0):
+        """ Initializing with GraphSON version 3.0
+
+        Args:
+            version (int):
+        """
+
+        self.graphsonVersion = version
+        pass
+
+    def connect(self, host="localhost", port="8182", graph="g", **kwargs):
+        """ Connect to JanusGraph's gremlin-server instance. Takes URL, Port and Graph
```

Comment:
First, it doesn't take an URL, but a host. An URL would be `ws://localhost:8182/gremlin` which wouldn't work.
More importantly, `g` doesn't represent the `graph`, but just a graph traversal source. Naming this parameter `graph` is really confusing. (Refer to the TinkerPop docs for an explanation of what a graph traversal source is.)

Reply:
Type. Also I've updated from  `graph` to `traversal_source` to keep it analogous with gremlin-python.